src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -219,30 +226,29 @@ public SVCustomReadFilter(final UseEnd whichEnd,
                                   final boolean filterDupReads,
                                   final boolean filterSecondaryAlignments,
                                   final boolean filterSupplementaryAlignments,
-                                  final int MQThreshold){
+                                  final int MQThreshold,
+                                  final int mMQThreshold){
 
-                final UseEnd endVal = whichEnd;
+            final UseEnd endVal = whichEnd;
 
-                ReadFilter tempFilter = read -> 0!=read.getFragmentLength();
-                           tempFilter = tempFilter.and(read -> read.isPaired());
-                           tempFilter = tempFilter.and(read -> endVal == (read.isFirstOfPair() ? UseEnd.FIRST : UseEnd.SECOND));
+            ReadFilter tempFilter = read -> 0!=read.getFragmentLength();
+                       tempFilter = tempFilter.and(read -> read.isPaired());
+                       tempFilter = tempFilter.and(read -> endVal == (read.isFirstOfPair() ? UseEnd.FIRST : UseEnd.SECOND));
 
-                if(filterNonProperlyPairdReads)   { tempFilter = tempFilter.and(read -> read.isProperlyPaired());}
```

Comment:
typo: "filterNonProperlyPairedReads"

Also, I'm not sure this is a good filter to include - we want to recompute the ISD in this tool and filtering reads by this flag will bias our distribution, since aligners like BWA are already marking things as not properly paired if their insert size is too large. At the least I think the tool should default to including reads even if they don't have the "properly paired" set.